src/main/java/ru/hogwarts/school_re/service/impl/AvatarServiceImpl.java

```diff
@@ -24,6 +26,8 @@
 
     private final AvatarRepository avatarRepository;
 
+    Logger logger = LoggerFactory.getLogger(AvatarServiceImpl.class);
```

Comment:
не забываем про модификаторы
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(AvatarServiceImpl.class);
```